src/Advection/vector_invariant_advection.jl

```diff
@@ -224,8 +224,8 @@ end
     Sζ = scheme.vorticity_stencil
 
     @inbounds v̂ = ℑxᶠᵃᵃ(i, j, k, grid, ℑyᵃᶜᵃ, Δx_qᶜᶠᶜ, v) / Δxᶠᶜᶜ(i, j, k, grid) 
-    ζᴸ =  _left_biased_interpolate_yᵃᶜᵃ(i, j, k, grid, scheme.vorticity_scheme, ζ₃ᶠᶠᶜ, Sζ, u, v)
-    ζᴿ = _right_biased_interpolate_yᵃᶜᵃ(i, j, k, grid, scheme.vorticity_scheme, ζ₃ᶠᶠᶜ, Sζ, u, v)
+    ζᴸ =  _left_biased_interpolate_yᵃᶜᵃ(i, j, k, grid, scheme.vorticity_scheme, Az_qᶠᶠᶜ, ζ₃ᶠᶠᶜ, Sζ, u, v) / Azᶜᶠᶜ(i, j, k, grid)
+    ζᴿ = _right_biased_interpolate_yᵃᶜᵃ(i, j, k, grid, scheme.vorticity_scheme, Az_qᶠᶠᶜ, ζ₃ᶠᶠᶜ, Sζ, u, v) / Azᶜᶠᶜ(i, j, k, grid)
```

Comment:
```suggestion
    ζᴸ =  _left_biased_interpolate_yᵃᶜᵃ(i, j, k, grid, scheme.vorticity_scheme, Az_qᶠᶠᶜ, ζ₃ᶠᶠᶜ, Sζ, u, v) / Azᶠᶜᶜ(i, j, k, grid)
    ζᴿ = _right_biased_interpolate_yᵃᶜᵃ(i, j, k, grid, scheme.vorticity_scheme, Az_qᶠᶠᶜ, ζ₃ᶠᶠᶜ, Sζ, u, v) / Azᶠᶜᶜ(i, j, k, grid)
```

i think this is a bug